docs/application/dotnet/guides/machine-learning/singleshot.md

```diff
@@ -0,0 +1,125 @@
+    </privileges>
+    ```
+
+<a name="manage"></a>
+## Managing Tensor Information
+
+In the example mentioned in this page, the MobileNet v1 model for TensorFlow Lite is used. This model is used for image classification. Its input data type is specify bit width of each Tensor and its input dimension is `3 X 224 X 224`. The output data type of the model is the same as the input datatype but the output dimension is `1001 X 1 X 1 X 1`.
+
+To configure the tensor information, you need to create a new instance of the `Tizen.MachineLearning.Inference.TensorsInfo` class. Then, you can add the tensor information such as datatype, dimension, and name (optional) as shown in the following code:
+
+```C#
+/* Input Dimension: 3 * 224 * 224 */
+TensorsInfo in_info = new TensorsInfo();
+in_info.AddTensorInfo(TensorType.UInt8, new int[4] { 3, 224, 224, 1 });
+
+/* Output Dimension: 1001 for classification */
+TensorsData out_info = new TensorsInfo();
+out_info.AddTensorInfo(TensorType.UInt8, new int[4] { 1001, 1, 1, 1 });
+```
+
+<a name="load"></a>
+## Loading Neural Network Model and Configuring Runtime Environment
+
+1. Since the model file is located in the resource directory of your own application, you need to get its absolute path:
+
+    ```C#
+    string ResourcePath = Tizen.Applications.Application.Current.DirectoryInfo.Resource;
+    string model_path = ResourcePath + "models/mobilenet_v1_1.0_224_quant.tflite";
+    ```
+
+
+2. You can load the neural network model from storage and configure a runtime environment with the `Tizen.MachineLearning.Inference.SingleShot` class. The first parameter is the absolute path to the neural network model file. The remaining two parameters are the input and the output `TensorsInfo` instances. If there is an invalid parameter, `ArgumentException` is raised.
+
+
+
+    ```C#
+    /* Create SingleShot instance with model information */
+    SingleShot single = new SingleShot(model_path, in_info, out_info);
+    ```
+
+<a name="invoke"></a>
+## Invoking Neural Network Model using Input Data
+
+To invoke the neural network model, you need to create the `Tizen.MachineLearning.Inference.TensorsData` instance to pass the input data of the model. You can add various types of tensor data, which are already specified in the `TensorInfo` instance. However, the maximum size of `TensorsData` is 16. If the limit is exceeded, then `IndexOutOfRangeException` is raised. Input data is passed in a byte array format, byte[].
+
+```C#
+/* Input data for test */
+byte[] in_buffer = new byte[3 * 224 * 224 * 1];
+
+/* Set the input tensor data */
+TensorsData in_data = in_info.GetTensorsData();
+in_data.SetTensorData(0, in_buffer);
+```
+
+After preparing the input data, you can invoke the model and get the inference output result. The `SingleShot.Invoke()` method gets the input data to be inferred as a parameter and returns the `Tizen.MachineLearning.InferenceTensorsData` instance, which contains the inference result.
```

Comment:
**Global Comment**:
It is recommended to use a colon (:) before a code block in Tizen documents.
Request you to add a colon before "result".

...contains the inference result:

Please consider this comment and fix it throughout the document.

Reply:
ACK!

Reply:
ACK!